src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/SourceTextExtensions.cs

```diff
@@ -121,21 +121,26 @@ public static bool NonWhitespaceContentEquals(this SourceText source, SourceText
             return i == source.Length && j == other.Length;
         }
 
-        public static int? GetFirstNonWhitespaceOffset(this SourceText source, TextSpan? span = null)
+        public static int? GetFirstNonWhitespaceOffset(this SourceText source, TextSpan? span, out int newLineCount)
```

Comment:
Why group the newLineCount with this method? It feels like a separate concern. Even though we'd be dupliating the logic of iterating the text span It'd be more fluent to have a separate CountNewlines equivalent method.

Reply:
💭 Given the number of uses, it doesn't seem like we save much by separating it. Only one use site doesn't need the additional parameter.